msm/src/mvlookup.rs

```diff
@@ -113,13 +113,13 @@ pub struct MVLookupWitness<F, ID: LookupTableID> {
 #[derive(Debug, Clone)]
 pub struct LookupProof<T, ID> {
     /// The multiplicity polynomials
-    pub(crate) m: Vec<T>,
+    pub(crate) m: BTreeMap<ID, T>,
```

Comment:
I wonder if having bit tree maps instead of hash maps means we no longer need the `from_u32` function for table ids?

Reply:
I meant btree maps, just autocorrect typo